pkg/azureconstants/azure_constants.go

```diff
@@ -68,6 +68,8 @@ const (
 	VolumeAttributePartition      = "partition"
 	WellKnownTopologyKey          = "topology.kubernetes.io/zone"
 	WriteAcceleratorEnabled       = "writeacceleratorenabled"
+	CreatedByTag                  = "created-by"
+	AzureDDTag                    = "kubernetes-azure-dd"
```

Comment:
There is a chance that a customer would like to take a previously created dynamic azure disk and use it as a static PV elsewhere. In this case the tag might be present even though it is being attached as a static PV. Is there a more reliable way of knowing it Like storage class being empty ?

Reply:
fixed by relying another field `RequestedSizeGib` which could only be created by dynamic PV